ui/next.config.js

```diff
@@ -1,10 +1,11 @@
 /** @type {import('next').NextConfig} */
+const isLocal = process.env.NODE_ENV === 'development';
 const CSPDirectives = [
     "default-src 'none';",
-    `script-src 'self' ${process.env.NODE_ENV === 'development' ? "'unsafe-eval'" : ''};`,
+    `script-src 'self' ${isLocal && "'unsafe-eval'"};`,
```

Comment:
Won't this output "false"?

Reply:
if `isLocal` is false that is

Reply:
Ah, that's right. I've changed it to use ternary operator like before in the latest push.